index.js

```diff
@@ -109,7 +110,7 @@ export async function startBackup ({ dataURL, s3Region, s3BucketName, s3AccessKe
  */
 async function * fetchCID (url) {
   const res = await fetch(url)
-  if (!res.ok || !res.body) throw new Error(`failed to fetch CIDs: ${url}`)
+  if (!res.ok || !res.body) throw new Error(`Backup:${url} failed to fetch CIDs`)
```

Comment:
I'd use `log` to log the error if it's important to preserve the prefix and leave the error message as is.

Reply:
Awesome, done!